api/ifc/ifc.go

```diff
@@ -28,12 +28,23 @@ type KvLoader interface {
 
 // Loader interface exposes methods to read bytes.
 type Loader interface {
+
+	// IsRemote returns whether this Loader was created from a url
+	IsRemote() bool
```

Comment:
I was thinking this would make future code easier to read. For example, `if ldr.IsRemote()` reads better than `if ldr.Repo() == ""`.

However, I really like your point. I'll collapse the 2 methods into `Repo() string, bool`, where the 1st argument is the repo and the second indicates whether there is a repo.

Reply:
Will address this in #4652